bin/gen_plots.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+from glob import glob
+from pathlib import Path
+
+import scanpy as sc
+
+
+def gen_plots() -> None:
+    # The only thing staged in by Nextflow should be the anndata file
+    # (extension .h5ad). glob for it and read
+    adata_path = glob('**/*.h5ad', recursive=True)[0]
+    adata = sc.read_h5ad(adata_path)
+
+    # Establish plots directory and set the figure directory in sc
+    # settings
+    plots_dir = Path('plots')
+    plots_dir.mkdir()
+    sc.settings.figdir = plots_dir
+
+    # Define a mapping between different plot types and gene types
+    plot_colors = {
+        'total_counts': ('hemoglobin', 'sex_linked', 'stress_response'),
+        'pct_counts': ('mitochondrial', 'ribosomal', 'cell_cycle'),
+    }
+
+    # Generate the plots and save
+    # for plot_type, gene_types in plot_colors.items():
```

Comment:
Should this be commented out?

Reply:
nope, i had it commented out because it doesn't work for the `tiny` dataset because none of the genes in that data end up being annotated. As a result, none of the strings generated actually exist as keys in `adata` so it just gives an error. i'll think of a more dynamic way to do this so that it won't throw an error, since people might be using their own gene annotations that don't have these gene types